src/main/java/com/softserve/edu/greencity/ui/api/google/sheets/ValueProvider.java

```diff
@@ -95,8 +95,8 @@ private void logGetterInfo(Method method){
     public static Boolean remote() {
         try {
             logger.info("remote: " + Boolean.valueOf((String) getRow(25).get(1)));
-        return Boolean.valueOf((String) getRow(25).get(1));
-        //return false;
+        //return Boolean.valueOf((String) getRow(25).get(1));
+        return false;
```

Comment:
Don't forget to return as it was after you finish your work

Reply:
Yes, know this.